<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadDemo</class>
 <widget class="QMainWindow" name="ThreadDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtThreadDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="text">
         <string>Event A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Event B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Event C</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ThreadForm.qrc"/>
 </resources>
 <connections/>
</ui>
